packages/signature-v4/src/getCanonicalQuery.spec.ts

```diff
@@ -96,6 +96,17 @@ describe("getCanonicalQuery", () => {
     ).toBe("%F0%9F%90%8E=%F0%9F%92%A9&%F0%9F%90%8E=%F0%9F%A6%84");
   });
 
+  it("should sort URI-encode members of query param arrays", () => {
+    expect(
+      getCanonicalQuery(
+        new HttpRequest({
+          ...httpRequestOptions,
+          query: { "p": ["a", "à"] },
```

Comment:
`yarn format` is only complaining about this:
```suggestion
          query: { p: ["a", "à"] },
```